pom.xml

```diff
@@ -7,7 +7,7 @@
 
   <groupId>ome</groupId>
   <artifactId>bio-formats-documentation</artifactId>
-  <version>5.8.1-SNAPSHOT</version>
+  <version>5.8.1</version>
```

Comment:
Should this `version` be bumped to `5.8.2-SNAPSHOT` on master?

Reply:
Yes, definitely now that 5.8.1 has been tagged.